### PR TITLE
Add azure cli and aks kubectl builders

### PR DESCRIPTION
**Pull request by scotth1**
Cloud Build steps for az and az-kubectl.

### CODE REVIEW

**Comment by edu-shamilton1** [comment]
I can confirm consent.
I was an idiot and forgot to change user.email before committing code from a different account by accident.

File: az-kubectl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/learning-resources-hub/az
```
Comment (nof20) [inline az-kubectl/Dockerfile]: Can you explain who owns this project/container and how it's maintained?  It's opaque here & we're inviting people to build dependencies on it.

Is there an official image maintained by MS on MCR (like the Azure example below)?

Comment (scotth1) [reply az-kubectl/Dockerfile]: Ah, that should be in the same project, there are two steps built, first one is $PROJECT_ID/az and the second one is $PROJECT_ID/az-kubectl which is built FROM the first.
Looks like I forgot undo all the project specific things, I'll make a new commit.

**Comment by nof20** [comment]
Looks good.  Thanks!

**Pull request merged**